Libraries/LibJS/Runtime/DateConstructor.cpp

```diff
@@ -46,6 +46,12 @@ DateConstructor::~DateConstructor()
 }
 
 Value DateConstructor::call(Interpreter& interpreter)
+{
+    auto* date = static_cast<Date*>(construct(interpreter).as_object());
+    return js_string(interpreter.heap(), date->string());
+}
+
+Value DateConstructor::construct(Interpreter& interpreter)
```

Comment:
"Construct" means something very different in Serenity already (`Core::Object::construct()`) 🙁

Reply:
Yeah it's a bit unfortunate that the terms overlap, but I think there's high value in using the same terminology as the ECMAScript spec. :)